include/lsst/afw/geom/SpherePoint.h

```diff
@@ -265,9 +268,6 @@ class SpherePoint final {
      * @param other the point to which to measure the bearing
      * @returns the direction, as defined above, in the interval [0, 2&pi;).
      *
-     * @throws pex::exceptions::DomainError
-     *         Thrown if `this.atPole()`.
-     *
```

Comment:
Please add a paragraph documenting the behavior at the poles. The current description implies that the bearing must always be -π/2 at the north pole and π/2 at the south pole, which I suspect is no longer what happens.

Reply:
OK. I rewrote the description to one whose behavior is clearer at the poles.